app/controllers/api/v0/pages_controller.rb

```diff
@@ -17,14 +17,7 @@ def index
     page_ids = id_query.pluck(:uuid, *order_attributes).collect {|data| data[0]}
 
     result_data =
-      if should_include_versions
-        results = query
-          .where(uuid: page_ids)
-          .includes(:versions)
-          .where(versions: { different: true })
-          .order('versions.capture_time DESC')
-        lightweight_query(results, &method(:format_page_json))
-      elsif should_include_latest
+      if should_include_latest
```

Comment:
You need to keep the `if should_include_versions` branch here so you can throw an error!

Additionally, I think we need to keep the code in place and determine whether to throw an error or run it like we used to based on configuration. (Maybe `ALLOW_VERSIONS_IN_PAGE_RESPONSES`?) This is a pretty dangerous change, and if we find out that some other important service is relying on it in a way we didn’t know about, we need to be able to quickly turn it back on.